sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import uuid
+from datetime import datetime
+from ._generated import AzureDigitalTwinsAPI
+
+class DigitalTwinsClient(object):
+    """Creates an instance of AzureDigitalTwinsAPI.
+
+    :param endpoint: The URL endpoint of an Azure search service
+    :type endpoint: str
+    :param credential: A credential to authenticate requests to the service
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    """
+    def __init__(self, endpoint, credential, **kwargs: any):
+        # type: (str, AzureKeyCredential, **Any) -> None
+
+        self.endpoint = endpoint #type: str
+        self.credential = credential #type AzureKeyCredential
+        self._client = AzureDigitalTwinsAPI(
+            credential=credential,
+            base_url=endpoint,
+            **kwargs
+        ) #type: AzureDigitalTwinsAPI
+
+    def get_digital_twin(self, digital_twin_id, **kwargs):
+        # type: (str, **Any) -> object
+        """Get a digital twin.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :returns: The twin object.
+        :rtype: object
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.get_by_id(digital_twin_id, **kwargs)
+
+    def upsert_digital_twin(self, digital_twin_id, digital_twin, **kwargs):
+        # type: (str, object, **Any) -> object
+        """Create or update a digital twin.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        412 (Precondition Failed): The model is decommissioned.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param digital_twin: The twin object to create or update.
+        :type digital_twin: object
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
```

Comment:
We should avoid documenting parameters as HTTP headers - rather in terms of how customers would use it.
What alternatives can a user supply apart from JSON?

Reply:
You are right, there is no other value is accepted. Removed.